### DISCUSSION TITLE
Quick Started | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1826] [Q&A]
# Quick Started | Seeed Studio Wiki

Get_Started_with_SenseCAP_T1000_tracker

https://wiki.seeedstudio.com/Get_Started_with_SenseCAP_T1000_tracker/

**Comment by tytoalba99** [comment]
About Data Cache:

I did a small trip (walking about one hour) with my T-1000 A but I'm not able to retrieve offline geolocations. When is the data uplinked ?
